packages/postgresql/src/Adaptor.js

```diff
@@ -234,12 +240,16 @@ export function findValue(filter) {
             if (result.rows.length > 0) {
               returnValue = result.rows[0][uuid];
             }
-            resolve(returnValue);
+            const nextState = {
+              ...composeNextState(state, returnValue),
+              result: returnValue,
+            };
+            resolve(nextState);
           }
         });
       });
     } catch (e) {
-      client.end();
+      console.log(e, 'asd');
```

Comment:
This logging needs removing. And are you sure you don't want to end the client here?

Reply:
Wait does this even fire?

Ach, I don't want to touch code I don't understand if I don't need to..

Reply:
Removed the `console.log()`

Reply:
No worries, I do that sort of thing all the time :) This is why we self-review